test/kitchen/site-cookbooks/dd-system-probe-check/recipes/linux.rb

```diff
@@ -42,8 +42,22 @@
   case node[:platform]
   when 'redhat', 'centos', 'fedora', 'amazon'
     package_name 'java'
-  when 'ubuntu', 'debian'
+  when 'ubuntu'
+    package_name 'default-jre'
+  when 'debian'
     package_name 'default-jre'
+    # ignore failure because of error about /etc/ssl/certs/java/cacerts
+    ignore_failure true
```

Comment:
i don't like it, as it can "swallow" other problems with java package.
is there anything on the internet with `ca-certificates-java` and debian?

Reply:
It doesn't swallow errors, because the next install block doesn't ignore failures.

Reply:
Because the problem is with the package itself, it is really hard to workaround. I tried many things:
1. installing in a different order, didn't matter because `ca-certificates-java` pulls in `default-jre-headless` regardless of recommends flag.
2. Created directories/files ahead of time, didn't fix it

The root issue was fixed, but not for debian 10 because it is too old (oldoldstable at this point).